### PR TITLE
[BUGFIX] Safety check for foreach

### PR DESCRIPTION
**Pull request by sabbelasichon**
Resolves: #21

### CODE REVIEW

File: src/ClassAliasLoader.php [hunk 1]
```diff
@@ -172,9 +172,11 @@ protected function getOriginalClassName($aliasOrClassName)
     protected function loadOriginalClassAndSetAliases($originalClassName)
     {
         if ($this->classOrInterfaceExists($originalClassName) || $this->loadClass($originalClassName)) {
-            foreach ($this->aliasMap['classNameToAliasMapping'][$originalClassName] as $aliasClassName) {
-                if (!$this->classOrInterfaceExists($aliasClassName)) {
-                    class_alias($originalClassName, $aliasClassName);
+            if(is_array($this->aliasMap['classNameToAliasMapping'][$originalClassName])) {
```
Comment (bmack) [inline src/ClassAliasLoader.php:175]: one minor remark: Can you add a space between the "if" and the "(" so it is consistent? We badly need some CI checks in this package ;)

Comment (sabbelasichon) [reply src/ClassAliasLoader.php]: Done.

**Comment by sbuerk** [comment]
@bmack @helhum This PR would also help to fix a functional test failure in core v11 with PHP 8.2. From myside, the change seems to be ok.

Any change to get this "merged" and a "release" so we can raise this package in TYPO3 core ? (If nothing speaks against it)

**Review by sbuerk — APPROVED** [review]

**Review by oliverklee — APPROVED** [review]

**Review by lolli42 — APPROVED** [review]

**Comment by helhum** [comment]
> Could it be that this would be fixed with #20 as well?

Fixed with #20

**Comment by hdietrich-timespin** [comment]
> 



> > Could it be that this would be fixed with #20 as well?
> 
> Fixed with #20

No is not fixed. 

The function TYPO3\ClassAliasLoader::loadOriginalClassAndSetAliases($originalClassName) don't use TYPO3\ClassAliasLoader::getOriginalClassName($aliasOrClassName). And so the Issue #22  is not solved!

**Pull request closed without merging**